### ISSUE TITLE
Build a Classification Model  for COVID-19 Radiography Dataset

### ISSUE THREAD

**Issue by arnab031** [CoinShift-Imaging/CoinShift-Imaging-Box#24]
**Statement.**
Here a dataset has collected from kaggle. Here is the link - https://www.kaggle.com/tawsifurrahman/covid19-radiography-database . Here we get different images of  X-Ray. We can say from X-Ray is it Covid-19 or Viral Pneumonia or Normal fever.

**Solution**
Here we build a model for classification using Pytorch. Afterwards we will calculate validation accuracy for test dataset and also plot graph of it.

**Additional**
We can compare the training time between  a normal deep learning model and transfer learning model also.

**Comment by arnab031** [comment]
As a GSSOC21 Participant , I would like to work on this issue. @DARK-art108

**Comment by Rishikeshrajrxl** [comment]
@arnab031, go ahead.

**Comment by DARK-art108** [comment]
@arnab031 Add one Directory of Cllasification_Zoo, Where we can add all solutions for the classification tasks.
